packages/react-examples/.storybook/preview.js

```diff
@@ -77,53 +78,47 @@ function loadStories() {
 }
 
 /**
+ * Generate stories (usually one story) for a `*.Example.tsx` or `*.stories.tsx` file
  * @param {string} key - key for the module in require.context (the relative path to the module
  * from the root path passed to require.context)
- * @param {Map<string, Story>} stories
+ * @param {Map<string, StoryGroup>} stories - mapping from component path (`Components/<ComponentName>`) to stories
  * @param {__WebpackModuleApi.RequireContext} req
  */
-function generateStoriesFromExamples(key, stories, req) {
-  // Depending on the starting point of the context, and the package layout, the key will be like one of these:
+function generateStoriesForExampleFile(key, stories, req) {
+  // The key will be like this:
   //   ./ComponentName/ComponentName.Something.Example.tsx
```

Comment:
With e2e and dynamic dependency story inclusion removed, the key should only have one structure :tada: